support-frontend/test/selenium/util/IdapiBypassRecaptchaRequest.scala

```diff
@@ -0,0 +1,54 @@
+package selenium.util
+
+import okhttp3.{OkHttpClient, Request, RequestBody, MediaType, Response}
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.{Json, Decoder, DecodingFailure}
+import io.circe.parser.parse
+
+import org.slf4j.LoggerFactory
+
+object IdapiBypassRecaptchaRequest {
+
+  private val client = new OkHttpClient()
+
+  private def logger = LoggerFactory.getLogger(this.getClass)
+
+  private val request: Request = new Request.Builder()
+    .url(Config.idapiNewTestUserUrl)
+    .addHeader(Config.idapiClientAccessTokenName, Config.idapiClientAccessTokenSecret)
+    .post(RequestBody.create(MediaType.parse("application/json"), "{}"))
+    .build()
+
+  private def getResult(): Either[String, Json] = {
+    val response: Response = client.newCall(request).execute()
+    val body: String = response.body().string()
+    parse(body) match {
+      case Left(error) => Left(s"Could not parse $body as JSON from response with code ${response.code()}: $error")
+      case Right(json) => Right(json)
+    }
```

Comment:
if you're only changing the left you can use leftMap
```suggestion
    parse(body).leftMap { error =>
      s"Could not parse $body as JSON from response with code ${response.code()}: $error"
    }
```

Reply:
Nice!

Reply:
Added